dvc/stage.py

```diff
@@ -206,14 +206,14 @@ def changed_md5(self):
     @property
     def is_callback(self):
         """
-        A callback stage is always considered as changed,
+        An orphan stage is always considered as changed,
```

Comment:
I'm not sure this is the correct term. It's not necessarily orphaned.

Reply:
Kk, reverted after long discussion on Discord. I still don't understand what a callback stage is though.